experimenter/experimenter/experiments/constants.py

```diff
@@ -505,3 +506,82 @@ class EmailType(models.TextChoices):
     ERROR_MULTIFEATURE_TOO_MANY_FEATURES = (
         "Multi-feature experiments can only support up to 20 different features."
     )
+
+    class ChangeEvent(Enum):
```

Comment:
We can get this information programattically. We don't need to define this structure.